de.metas.adempiere.adempiere/base/src/test/java/org/adempiere/location/geocoding/openstreetmap/NominatimOSMGeoCoordinatesProviderImplTest.java

```diff
@@ -0,0 +1,156 @@
+package org.adempiere.location.geocoding.openstreetmap;
+
+import org.adempiere.location.geocoding.GeographicalCoordinates;
+import org.adempiere.location.geocoding.GeoCoordinatesRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/*
+ * #%L
+ * metasfresh-pharma
+ * %%
+ * Copyright (C) 2018 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+class NominatimOSMGeoCoordinatesProviderImplTest
+{
+	private NominatimOSMGeoCoordinatesProviderImpl coordinatesProvider;
```

Comment:
these test will hit the OSM-API on each build, right? wdyt about disabling it and keep it only for devs to run them manually?

Reply:
Done